src/main/java/org/cactoos/text/Split.java

```diff
@@ -45,6 +45,11 @@
      */
     private final UncheckedText regex;
 
+    /**
+     * The limit.
```

Comment:
@victornoel We need more documentation on how this `limit` works: how is the expected behavior with different limit values (> 0, == 0 and <0)